app/src/main/java/com/monkeyteam/chimpagne/viewmodels/EventViewModel.kt

```diff
@@ -402,6 +407,65 @@ class EventViewModel(
         {})
   }
 
+  fun createAPollAtomically(

```

Comment:
As discussed previously, you should not put articles in function names

Reply:
Indeed I did not follow the conventions, it is fixed now.